### PR TITLE
Fix synchronous99

### PR DESCRIPTION
**Pull request by HansOlsson**
The remaining synchronous decisions from Design 99 - and some image cropping.
They might need a bit of further refinement.

@svorkoetter this might be something for you or @eshmoylova

### CODE REVIEW

**Comment by HansOlsson** [comment]
@sjoelund Can you review it, or know who can?

**Comment by sjoelund** [comment]
I hope Bernhard Thiele could check it. I didn't see anything wrong in the text, but I have not touched the synchronous parts in the compiler or used it in my models...

File: chapters/synchronous.tex [hunk 1]
```diff
@@ -608,12 +612,14 @@ \subsection{Base-clock conversion operators}\doublelabel{base-clock-conversion-o
...
 \begin{longtable}[]{|l|p{12cm}|}
 \hline \endhead
-\textbf{sample}(u, c) &
+\textbf{sample}(u, clock) &
 Input argument u is a continuous-time expression according to
-\autoref{continuous-time-expressions}. The optional input argument c is of type Clock. The operator
-returns a clocked variable that has c as associated clock and has the
-value of the left limit of u when c is active (that is the value of u
-just before the event of c is triggered). If argument c is not provided,
+\autoref{continuous-time-expressions}. The optional input argument clock is of type Clock, and can in a call be given as a named argument (with the name clock),
```
Comment (miscco) [inline chapters/synchronous.tex]: There are some really strange line breaks here

File: chapters/synchronous.tex [hunk 2]
```diff
@@ -737,18 +753,11 @@ \subsection{Sub-clock conversion operators}\doublelabel{sub-clock-conversion-ope
...
 shiftCounter, resolution)
 \end{tabular}
-& {[}\emph{The first activation of the clock of y =
-\textbf{shiftSample}(..) is shifted in time
-shiftCounter/resolution*interval(u) later than the first activation of
-the clock of u.}{]}.
-
-Conceptually, the operator constructs a clock ``cBase''
+& The operator \lstinline!shiftSample(u,k)! skips the first k ticks of u, and is then returning every tick.
+With resolution it is defined as:
```
Comment (miscco) [inline chapters/synchronous.tex]: I think this is not sufficient. The fractional shifts of wrt resolution are really non trivial. We should find a prosa expression for that too. Maybe something along the lines of:

>The operator \lstinline! c=shiftSample(u,k,resolution)! shifts the first activation of the clock c by `k * interval(u) / resolution` relative to the first activation of `u`.  The clock `c` is then activating with the same frequency as `u`.

Comment (miscco) [reply chapters/synchronous.tex]: Maybe even better:

> The operator \lstinline! c=shiftSample(u,k,resolution)! splits the interval between activations of the clock of u into `resolution` equidistant intervals `i`. The clock `c` then activates `k` intervals `i` after each activation of `u`.

Comment (HansOlsson) [reply chapters/synchronous.tex]: Looks good, but I don't like having two normative explanations for the same thing.
Should we have this as non-normative or the equation-form as non-normative?

File: chapters/synchronous.tex (hunk 1 above)
Comment (HansOlsson) [reply chapters/synchronous.tex]: True, but it works and can clean up later.

**Comment by miscco** [comment]
I am happy with it?!

**Pull request merged**